nav2_amcl/include/nav2_amcl/pf/pf_pdf.hpp

```diff
@@ -30,9 +30,6 @@
 
 #include "nav2_amcl/pf/pf_vector.hpp"
 
-// #include <gsl/gsl_rng.h>
```

Comment:
Why add?

Reply:
Basically because this code has been commented out since 2018; I think it is time to remove it :).  Anyone trying to resurrect it will probably have to reimplement a bunch of stuff anyway, so I don't particularly see a need to keep it around anymore.

That said, if you'd like to discuss it separately I'm happy to put it into a separate PR.

Reply:
I suppose I'd rather keep all the code that contained AMCL intact for whoever might come along and care to really redevelop it. I can't gauge from a glance how useful / important any of those utilities or old methods are, so I'd prefer to keep them commented out than deleted so that there's an explicit historical record of it. AMCL and Costmap2D are two "special" packages in my book that get a bit of different treatment for code treatment since there's alot of history there, much of which I know, but I wasn't there and I don't know when things like this might be back to be helpful in the future.

Reply:
All right, fair enough.  I've just rebased onto the latest and taken out the changes that removed the dead code.